pom.xml

```diff
@@ -66,6 +66,13 @@
             <version>${version.slf4j}</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>jsr305</artifactId>
+            <version>3.0.2</version>
+            <optional>true</optional>
```

Comment:
Note this is an optional dependency addition so it won't impact downstream consumers.

Reply:
Fair enough :)